modules/cleaning-crio-storage.adoc

```diff
@@ -69,6 +69,15 @@ $ sudo -i
 ----
 +
 . Stop the containers and pods:
+
+.. Use the following command to stop the pods that are not in the `HostNetwork`. They must be removed first, because their removal relies on the networking plugin pods, which are in the `HostNetwork`.
```

Comment:
```suggestion
.. Use the following command to stop the pods that are not in the `HostNetwork`. They must be removed first because their removal relies on the networking plug-in pods, which are in the `HostNetwork`.
```

Reply:
fixed!